backend/tests/integration/tests/test_inventory.py

```diff
@@ -312,11 +312,7 @@ def add_devices_to_tenant(tenant, dev_inventories):
 
     return tenant
 
-
-@pytest.mark.skipif(
-    useExistingTenant(), reason="not feasible to test with existing tenant",
-)
-class TestDeviceFilteringEnterprise:
+class TestDeviceFilteringBase:
```

Comment:
Since pytest will collect all classes with `Test` prefix, it might be a good idea to use a different class name?
```suggestion
class DeviceFilteringTests:
```

Reply:
Yep, that's a good idea indeed! 🤦 Nice catch - fixed in 4f33330620b8c3896b1a5484a01257d3c248082e